src/interfaces/phase_integrals.jl

```diff
@@ -0,0 +1,75 @@
+####################
+# The abstract phase integral interface
+#
+# In this file, the abstract interface for different copmutation methods of
+# phase integrals is defined.
+####################
+"""
+Abstract base type for defining a method for phase integral computation.
+"""
+abstract type Method end
```

Comment:
Please add the type/struct/function definition (or something close that gets the idea across) in the docs too.
```suggestion
"""
    abstract type Method end

Abstract base type for defining a method for phase integral computation.
"""
abstract type Method end
```

Also I'm not sure if the names `Method` and `Analytic` are a bit too unspecialized.